.github/workflows/integration-test.yml

```diff
@@ -1,26 +1,18 @@
 name: Integration Tests
 
-on:
```

Comment:
This section needs to be kept, but the filters should be removed, it could look like this:

```
on: [push, pull_request]
```
https://github.com/opensearch-project/security/blob/main/.github/workflows/ci.yml#L3